Enable building with g++ 9.x

Disable -Werror for new warning "deprecated-copy" to avoid build
failure in the packaged copy of googletest with g++ 9.x.  This is
hopefully temporary until googletest is upgraded or it becomes
straightforward to build against an externally provided copy.

Upstream-Status: pending

Signed-off-by: Scott Murray <scott.murray@konsulko.com>

diff --git a/ThirdParty/googletest-release-1.8.0/googletest/cmake/internal_utils.cmake b/ThirdParty/googletest-release-1.8.0/googletest/cmake/internal_utils.cmake
index 777b91e..7832f20 100644
--- a/ThirdParty/googletest-release-1.8.0/googletest/cmake/internal_utils.cmake
+++ b/ThirdParty/googletest-release-1.8.0/googletest/cmake/internal_utils.cmake
@@ -91,7 +91,7 @@ macro(config_compiler_and_linker)
     set(cxx_no_exception_flags "-D_HAS_EXCEPTIONS=0")
     set(cxx_no_rtti_flags "-GR-")
   elseif (CMAKE_COMPILER_IS_GNUCXX)
-    set(cxx_base_flags "-Wall -Wshadow")
+    set(cxx_base_flags "-Wall -Wshadow -Wno-error=deprecated-copy")
     set(cxx_exception_flags "-fexceptions")
     set(cxx_no_exception_flags "-fno-exceptions")
     # Until version 4.3.2, GCC doesn't define a macro to indicate
